### ISSUE TITLE
Univariate time series Problem

### ISSUE THREAD

**Issue by mkozkok** [hyunsooseol/snowCluster#15]
Hi, 
I want to use ARIMA or Prophet module for my population forecasting research. But when I add the variables, I receive this message. I can't find any "newbie-friendly" tutorial for this extension. Also, can you explain to me about using this extension?
My data (ss-1)
Error message (ss-2, ss-3)
My file (poppr.omv)

Best regards.

![ss-1](https://github.com/hyunsooseol/snowCluster/assets/138057925/bffb5287-a0ab-4c98-b786-4b170bb2bb83)
![ss-2](https://github.com/hyunsooseol/snowCluster/assets/138057925/d9f5956e-dbd9-4ddc-b55d-b10a318a9d72)
![ss-3](https://github.com/hyunsooseol/snowCluster/assets/138057925/13d9fa02-3814-4cc5-9c54-c385dafe67e0)
[poppr_.zip](https://github.com/hyunsooseol/snowCluster/files/11992539/poppr_.zip)

**Comment by hyunsooseol** [comment]
Hi

I think your data format seems to be wrong. please read instructions and check the your dataset.

Best

Seol


________________________

Hyunsoo Seol, Ph.D., Professor

Dept.of Education, Chung-Ang Univ.

c.p) 010-3294-5717, o) 820-5370

H) http://jamovi.kr<http://jamovi.kr/>

[cid:33e87dbe-52bd-44d5-b5a8-a21fd9ae227b]
________________________________
보낸 사람: mkozkok ***@***.***>
보낸 날짜: 2023년 7월 9일 일요일 오전 7:20
받는 사람: hyunsooseol/snowCluster ***@***.***>
참조: Subscribed ***@***.***>
제목: [hyunsooseol/snowCluster] Univariate time series Problem (Issue #15)


Hi,
I want to use ARIMA or Prophet module for my population forecasting research. But when I add the variables, I receive this message. I can't find any "newbie-friendly" tutorial for this extension. Also, can you explain to me about using this extension?
My data (ss-1)
Error message (ss-2, ss-3)

Best regards.

[ss-1]<https://user-images.githubusercontent.com/138057925/252069518-bffb5287-a0ab-4c98-b786-4b170bb2bb83.png>
[ss-2]<https://user-images.githubusercontent.com/138057925/252069525-d9f5956e-dbd9-4ddc-b55d-b10a318a9d72.png>
[ss-3]<https://user-images.githubusercontent.com/138057925/252069529-13d9fa02-3814-4cc5-9c54-c385dafe67e0.png>

**Comment by mkozkok** [comment]
Hi,
The data I want to use for time series analysis is population data with a one-year interval (2007-2022). I named the population column "y" and converted it to continuous data (ss-4). However, this time I get the error "time series has no or less than 2 periods". Can't we use one-year interval data? Should I use ARIMA or Prophet for this analysis?

![ss-4](https://github.com/hyunsooseol/snowCluster/assets/138057925/f4846274-03a8-4487-8e66-b3ee6ce60922)

**Comment by hyunsooseol** [comment]
Hi

I check for problems with the statistics module, but I don't do statistics consulting.

Best
Seol


________________________

Hyunsoo Seol, Ph.D., Professor

Dept.of Education, Chung-Ang Univ.

c.p) 010-3294-5717, o) 820-5370

H) http://jamovi.kr<http://jamovi.kr/>

[cid:b904c841-9474-4450-a901-7f9862a3b313]
________________________________
보낸 사람: mkozkok ***@***.***>
보낸 날짜: 2023년 7월 9일 일요일 오전 7:42
받는 사람: hyunsooseol/snowCluster ***@***.***>
참조: 설현수 ***@***.***>; Comment ***@***.***>
제목: Re: [hyunsooseol/snowCluster] Univariate time series Problem (Issue #15)


Hi,
The data I want to use for time series analysis is population data with a one-year interval (2007-2022). I named the population column "y" and converted it to continuous data (ss-4). However, this time I get the error "time series has no or less than 2 periods". Can't we use one-year interval data? Should I use ARIMA or Prophet for this analysis?

[ss-4]<https://user-images.githubusercontent.com/138057925/252074793-f4846274-03a8-4487-8e66-b3ee6ce60922.png>

**Comment by mkozkok** [comment]
Hi,
Now I tried using the PROPHET model. But I got a different error (ss-5, and zip file). I am using my computer in Turkish and Turkey time zone, is it because of this? How should I process time data?

![ss-5](https://github.com/hyunsooseol/snowCluster/assets/138057925/b8335b28-e7e0-4e53-9930-cf2515d66cea)
[poppr_proph.zip](https://github.com/hyunsooseol/snowCluster/files/11992580/poppr_proph.zip)

And my problem as I mentioned above still continues. The data I want to use for time series analysis is population data with a one-year interval (2007-2022). I named the population column "y" and converted it to continuous data (ss-4). However, this time I get the error "time series has no or less than 2 periods" in ARIMA.
